TechnicoBackend/Services/RepairService.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using TechnicoBackend.Models;
+using TechnicoBackend.Repositories;
+
+namespace TechnicoBackend.Services
+{
+    public class RepairService
```

Comment:
Consider using an interface IRepairService from which the current class would be inherited.